tiledb/sm/filesystem/vfs.cc

```diff
@@ -93,7 +93,7 @@ VFS::VFS(
 #endif
 
 #ifdef _WIN32
-  throw_if_not_ok(win_.init(config_));
+  win_ = Win(config_);
 #else
```

Comment:
After `config_` gets removed:

```suggestion
#ifndef _WIN32
```